### PR TITLE
document abra commands for managing secrets and for checking logs

### PR DESCRIPTION
**Pull request by dwaxweiler**
## What does this PR do?

I have added some abra commands for managing secrets and for checking logs, which I learned when deploying an own Karrot instance. Others might find them useful too.

## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [X] no unrelated changes
- [x] asked someone for a code review
- [ ] joined [#karrot:matrix.org](https://matrix.to/#/#karrot:matrix.org)
- [ ] added an entry to CHANGELOG.md (description, pull request link, username(s))
- [ ] tried out on a mobile device (does everything work as expected on a smaller screen?)

### CODE REVIEW

**Comment by dwaxweiler** [comment]
@nicksellen, could you review this please?

File: docs/src/self-hosting.md [hunk 1]
```diff
@@ -32,8 +32,22 @@ _Note: you might find some rough edges at this time, we are very interested to s
     - choose your server
     - choose a domain name that karrot will be deployed under: this is what we refer to as `karrot app name`
 2. `abra app config <karrot app name>` see [configuration options](#configuration-options) below
-3. `abra app deploy <karrot app name>`
-4. Wait for a few minutes while karrot initializes and then you can start creating accounts!
+3. Set all the secrets, even the ones you don't want to use:
+   - `abra app secret list <karrot app name>` for listing all secrets
+   - `abra app secret insert <karrot app name> <secret name> <version> "<secret value>"` for inserting a secret
+   - `abra app secret generate <karrot app name> <secret name> <version>` for generating a secret
```
Comment (nicksellen) [inline docs/src/self-hosting.md:38]: You can use `abra app secret general --all` (or short `-a`) for generating all the remaining secrets, so set the "external" ones first, then can use all to set the generated ones.

**Review by nicksellen — APPROVED** [review]
Great! Thanks for the contribution.

I added one optional comment you can choose to include or not.

File: docs/src/self-hosting.md (hunk 1 above)
Comment (dwaxweiler) [reply docs/src/self-hosting.md:38]: @nicksellen, there is one of the recently added secrets that did not accept the one generated with this command, but I don't remember which one. 🙈 Therefore, I would leave it out for now.

Comment (nicksellen) [reply docs/src/self-hosting.md:38]: Great, merged! One day I hope the secret management can know the difference between ones that have to be specifically inserted, and generated ones, ... and _how_ to generate specific secrets.

**Pull request merged**